food-bank-app/src/hooks/useClient/useClient.ts

```diff
@@ -0,0 +1,5 @@
+import HttpClient from './httpClient'
+
+const useClient = (): HttpClient => new HttpClient(process.env.API_URL)
```

Comment:
we could actually just import HttpClient on our views, but I created this hook just to do things the React way